exercises/ruby-1-b/README.md

```diff
@@ -4,17 +4,27 @@ This is Part 1 of our research into how people write Ruby differently. This exer
 
 ## Instructions
 
-Your task is to determine which word in a sentence has the most vowels (a, e, i, o, u). A sentence is defined as a [Ruby string][docs-string] consisting of one or more words. A word consists of one or more upper- or lowercase letters. Words are separated from each other by a single space character.
+You are given a collection of library books, and your job is to import them into
+the library's catalog system. The books are in the constant `Library::BOOKS`, and
+each book has the following columns:
+title, ISBN, author, and publishing year.
 
-Your job is to define a function named `word_with_most_vowels`, which takes a sentence, and returns the word with the most vowels.
+You need to write the logic for finding a book, adding new copies to the library's
+inventory, lending a book, and returning a book
 
-For example:
-
-```ruby
-word_with_most_vowels("Hi there aunt jeanie")
-// Returns: "jeanie"
+A book should be findable with it's ISBN
+```
+book = Library.new.find("9780191604928")
+```
+And it should have getters for the attribute values provided in `Library::BOOKS`
 ```
+book.author
+=> "Mark Twain"
```

Comment:
This is slightly more consistent with the [ruby-1-a README](https://github.com/exercism/research_experiment_1/tree/master/exercises/ruby-1-a):

```suggestion
// Returns: "Mark Twain"
```